src/libraries/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -252,26 +255,18 @@ System.Reflection.PortableExecutable.ManagedPEBuilder</PackageDescription>
     <Compile Include="System\Reflection\Throw.cs" />
     <Compile Include="System\Reflection\System.Reflection.cs" />
   </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="$(LibrariesProjectRoot)System.Collections.Immutable\src\System.Collections.Immutable.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+
+  <ItemGroup Condition="'$(TargetFrameworkVersion)' == 'v$(NetCoreAppCurrentVersion)'">
```

Comment:
Can this be

```
'$(TargetFramework)' == '$(NetCoreAppCurrent)'
```
?

Reply:
You are right. Let me update that line a follow-up PR.